.gitignore

```diff
@@ -25,3 +25,7 @@ web/update.php
 /vendor/
 /web/modules/contrib/
 /web/themes/contrib/
+
+/web/sites/default/services.yml
```

Comment:
We don't exactly understand why this change is part of this task and why it ignores a Drupal file.